shell/common/vsync_waiter_fallback.cc

```diff
@@ -27,6 +28,8 @@ VsyncWaiterFallback::~VsyncWaiterFallback() = default;
 
 // |VsyncWaiter|
 void VsyncWaiterFallback::AwaitVSync() {
+  TRACE_EVENT0("flutter", "VSYNC");
```

Comment:
Shouldn't this be a `TRACE_INSTANT0`. It is not meant to be a slice I don't think.

Reply:
I was following what's being done on: https://github.com/flutter/engine/blob/master/shell/platform/android/vsync_waiter_android.cc#L47 and https://github.com/flutter/engine/blob/master/shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm#L71 as I wasn't sure how catapult expects these. I will investigate changing it to an instant.